contentcuration/locale/CSV_FILES/contentcuration/locale/ar_SA/LC_FRONTEND_MESSAGES/contentcuration-messages.json

```diff
@@ -1212,5 +1212,6 @@
     "ForgotPassword.forgotPasswordFailed": "فشلت عملية إرسال رابط تفعيل جديد. يرجى المحاولة مجدداً.",
     "AdministrationAppError.unauthorizedDetails": "يجب أن تكون مسؤولاً في الاستوديو لتتمكن من عرض هذه الصفحة",
     "AdministrationIndex.channelsLabel": "القنوات",
-    "AdministrationIndex.usersLabel": "المستخدمون"
+    "AdministrationIndex.usersLabel": "المستخدمون",
```

Comment:
@bjester How come was this missing only in Arabic?

Reply:
The diff is only due to the trailing comma. I tried to place the new strings into the JSON files somewhat alphabetically, but this one appeared all out of order so I just put it at the bottom

Reply:
Gotcha, just checking 👍🏽